core/cmd/hoverfly/main.go

```diff
@@ -283,6 +282,8 @@ func main() {
 		requestCache = nil
 	}
 
+	cfg.DisableProxyAuthoriation = *disableProxyAuth
```

Comment:
So the two options are `proxy-auth` and `header-auth`. Does `proxy-auth` accept both `Proxy-Authorization` and `X-HOVERFLY-AUTHORIZATION`? If not, this means that switching between local instances of Hoverfly and remote instances would become more of a pain as you would have to change the authorization header being sent each time.